tests/setup/owasp_dependency_scan_faker_setup.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+from collections.abc import Sequence
+from typing import Any, cast
+
+from faker import Faker, Generator
+
+from tests.input_formats.owasp_deependency_scan.entities.generators.owasp_dependency_report_model_provider import (  # noqa: E501
+    OWASPDependencyReportModelProvider,
+)
+
+
+class OwaspFaker(Faker):
+    owasp_provider: OWASPDependencyReportModelProvider
```

Comment:
You don't need this class attribute.
The constructor can also be more basic since it is never executed anyways. See the other Fakers.